aten/src/ATen/native/native_functions.yaml

```diff
@@ -3357,11 +3357,11 @@
   dispatch:
     CUDA: _cslt_compress
 
-- func: _cslt_sparse_mm(Tensor compressed_A, Tensor dense_B, Tensor? bias=None, Tensor? alpha=None, ScalarType? out_dtype=None, bool transpose_result=False, int alg_id=0) -> Tensor
+- func: _cslt_sparse_mm(Tensor compressed_A, Tensor dense_B, Tensor? bias=None, Tensor? alpha=None, ScalarType? out_dtype=None, bool transpose_result=False, int alg_id=0, int split_k=1, bool split_k_one_kernel=True) -> Tensor
   dispatch:
     CUDA: _cslt_sparse_mm
 
-- func: _cslt_sparse_mm_search(Tensor compressed_A, Tensor dense_B, Tensor? bias=None, Tensor? alpha=None, ScalarType? out_dtype=None, bool transpose_result=False) -> int
+- func: _cslt_sparse_mm_search(Tensor compressed_A, Tensor dense_B, Tensor? bias=None, Tensor? alpha=None, ScalarType? out_dtype=None, bool transpose_result=False) -> (int, int, bool, int)
```

Comment:
I'm wondering if we should add a new `_cslt_sparse_mm_full_search`, just for forward compatibility.

Reply:
Should these functions live as ATen native functions?

Reply:
I'll see if I can add the new matmul search [here](https://github.com/pytorch/pytorch/blob/main/torch/csrc/cuda/shared/cusparselt.cpp) instead, and we can deprecate the old search function in a couple of weeks.